### PR TITLE
Abort batch transaction if trigger function fails.

### PR DESCRIPTION
**Pull request by loadzero**
If a trigger function inside a continuous trigger batch to be processed fails, abort the transaction and clean up.

### CODE REVIEW

File: src/test/regress/sql/cont_trigger.sql [hunk 1]
```diff
@@ -1,4 +1,6 @@
-CREATE CONTINUOUS VIEW cont_tg_cv AS SELECT count(*) FROM cont_tg_stream;
+select pg_sleep(2);
```
Comment (usmanm) [inline src/test/regress/sql/cont_trigger.sql]: Why is this needed?

**Comment by usmanm** [comment]
Looks good! If that sleep was added by accident, you can push this after removing it.

File: src/test/regress/sql/cont_trigger.sql (hunk 1 above)
Comment (loadzero) [inline src/test/regress/sql/cont_trigger.sql]: The trigger process isn't started until after that sleep. The sleep was there before, I just moved it to the start. A better solution might be to block on creating triggers until the trigger proc is up.
